src/Evolution/Systems/GeneralizedHarmonic/TagsDeclarations.hpp

```diff
@@ -0,0 +1,40 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+namespace GeneralizedHarmonic {
+template <size_t Dim>
+struct SpacetimeMetric;
```

Comment:
Why not call this `Psi`? (I'm not requesting a change, just asking for the reason :) )

Reply:
I think it is more clear to try to tag things with what they are instead of the symbol used to denote them whenever possible.  In the implementation we can use aliases with the symbols to make it easier to follow equations in papers.

Reply:
Then shouldn't Pi and Phi be named something else?